pjmedia/include/pjmedia/videodev.h

```diff
@@ -162,6 +162,30 @@ enum pjmedia_vid_dev_std_index
 };
 
 
+/**
+ * Enumeration of window fullscreen flags.
+ */
+typedef enum pjmedia_vid_dev_fullscreen
```

Comment:
Suggestion:
Change `pjmedia_vid_dev_fullscreen` enum to `pjmedia_vid_dev_fullscr_flag` or `pjmedia_vid_dev_fscreen_flag` or `pjmedia_vid_dev_fscr_flag`

Reply:
Okay. Perhaps `pjmedia_vid_dev_fullscreen_flag`? Just ~3 chars more :)

Reply:
you mean 5 more? ;) as long as it contains "flag", it's okay.